stl/src/stacktrace.cpp

```diff
@@ -85,49 +86,61 @@ namespace {
             debug_symbols = nullptr;
         }
 
+        if (dbgeng != nullptr) {
+            FreeLibrary(dbgeng);
+            dbgeng = nullptr;
+        }
+
         initialize_attempted = false;
     }
 
     bool try_initialize() {
         if (!initialize_attempted) {
-            // Deliberately not calling CoInitialize[Ex]
-            // DbgEng.h API works fine without it. COM initialization may have undesired interference with user's code
-
-            if (SUCCEEDED(DebugCreate(IID_IDebugClient, reinterpret_cast<void**>(&debug_client)))
-                && SUCCEEDED(debug_client->QueryInterface(IID_IDebugSymbols, reinterpret_cast<void**>(&debug_symbols)))
-                && SUCCEEDED(
-                    debug_client->QueryInterface(IID_IDebugControl, reinterpret_cast<void**>(&debug_control)))) {
-                attached = SUCCEEDED(debug_client->AttachProcess(
-                    0, GetCurrentProcessId(), DEBUG_ATTACH_NONINVASIVE | DEBUG_ATTACH_NONINVASIVE_NO_SUSPEND));
-                if (attached) {
-                    (void) debug_control->WaitForEvent(0, INFINITE);
+            dbgeng = LoadLibraryW(L"dbgeng.dll");

```

Comment:
As `dbgeng.dll` lives in system32, I believe we need [`LoadLibraryExW()`](https://docs.microsoft.com/en-us/windows/win32/api/libloaderapi/nf-libloaderapi-loadlibraryexw) with  `LOAD_LIBRARY_SEARCH_SYSTEM32`:

https://github.com/microsoft/STL/blob/60decd0d829e68666b556780002c83605d748d80/stl/src/tzdb.cpp#L66

This avoids scary DLL planting issues.